montecarlo/stopping_condition.go

```diff
@@ -99,13 +99,34 @@ func (s *Simmer) shouldStop(iterationCount uint64,
 		// play also has no losing chance
 		tiebreakByEquity = true
 	}
+	tentativeWinner.RUnlock()
 
 	if tiebreakByEquity {
+		// We may need to re-determine the tentative winner.
+		highestEquity := -1000000.0
+		highestEquityIdx := -1
+		for idx, p := range c {
+			p.RLock()
```

Comment:
does this work, or should `p.RLock()` be `c[idx].RLock()`?

Reply:
`p` is a pointer in this case. maybe `c[idx].RLock()` reads better anyway